CHANGES.txt

```diff
@@ -3,6 +3,7 @@
 
 * Issue #1444: Correct typos in ``@cherrypy.expose``
   decorators.
+* #1441: Added tool to automatically convert request params.
```

Comment:
Thanks for the changelog entry. Can you move it into a new section with an 0.1 bump?